azavea/iow-boundary-tool
# Create `ReferenceImage` serializers and views

## Overview

Create the views and serializers for ReferenceImage (everything but the image data itself, which will be in #81)

Closes #88

### Demo

Use Django admin to add a bunch of reference images to the active submission for the test utility, then visit:
Browsable APIs:
http://localhost:8181/api/boundary/1/reference-images/


### Notes
I moved the test setup script out of resetdb into `tests.py` where we can actually start using it. I had talked before about having a minimal set of users for manual testing versus a larger suite of users in python objects, but on second look, I don't see the harm in using the same set.

## Testing Instructions

- `./scripts/manage resetdb`
- Login in as contributor
- Visit API endpoints at localhost:8181/api/boundaries/pk/reference-images and localhost:8181/api/boundaries/pk/reference-images/pk
- Repeat as admin
- Repeat as validator, ensure permission denied
- Repeat with invalid data, ensure errors caught

## Checklist
